website/docs/d/plugin.html.markdown

```diff
@@ -21,27 +21,6 @@ data "newrelic_plugin" "foo" {
   guid = "com.example.my-plugin"
 }
 
```

Comment:
Add an EOL message to this data source similar to the one we added for the resource?